slither/detectors/naming_convention/naming_convention.py

```diff
@@ -80,8 +90,11 @@ def detect(self):
                                     'sourceMapping': func.source_mapping})
 
                 for argument in func.parameters:
-
-                    if self.is_mixed_case(argument.name) is False:
+                    if argument in func.variables_read_or_written:
+                        incorrect_naming = self.is_mixed_case(argument.name) is False
+                    else:
+                        incorrect_naming = self.is_mixed_case_with_underscore(argument.name) is False
+                    if incorrect_naming:
```

Comment:
the `incorrect_naming` and `is False` is tricky/not so easy to understand.

Maybe we should revert it and make it a `correct_name`? This way we won't have `is False` twice and we will just do at the end `if not correct_name:`